R/check_result.R

```diff
@@ -14,6 +14,10 @@
 #'   where message is the matched result message.
 #' @template grader_args
 #' @template learnr_args
+#' @param glue_correct A glue string that returns the final correct message displayed.
+#'    Defaults to getOption("gradethis_glue_correct").
+#' @param glue_incorrect A glue string that returns the final incorrect message displayed.
```

Comment:
make template to use in other functions

Reply:
yes. was going to do that after i updated the params in check_code